Sanity/mariadb-odbc/main.fmf

```diff
@@ -0,0 +1,23 @@
+summary: Basic smoke-test for accessing MariaDb via unixODBC
+contact:
+- Filip Janus <fjanus@redhat.com>
+component:
+- python3-pyodbc
+- unixODBC
+- mariadb-connector-odbc
+- mariadb
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- python3-pyodbc
+- unixODBC
+- unixODBC-libs
+- mariadb-connector-odbc
+- mariadb-server
+duration: 5m
+extra-summary: 'TestVersion:     2.0'
```

Comment:
Just nit pick. Why so many white spaces?

Reply:
result of tmt test convert, didn't fixed it manually, as it is a string and might have a purpose (I do not think that either...)